StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+DeferredAPI.swift

```diff
@@ -167,4 +169,9 @@ extension PaymentSheet {
         }
         return paymentUserAgentValues
     }
+
+    private static func setAsDefaultErrorOnSSC(testMode: Bool, allowsSetAsDefaultPM: Bool) {
```

Comment:
this name confuses me. The verb (what this method does) is "set as default", but I think this method validates something and should be called, like, `validateSomethingSomething`?

Reply:
Or just inline this, not sure the method adds much value.  I'd also just combine the implementation into a single assert().